src/main/java/com/vmware/vipclient/i18n/base/instances/TranslationMessage.java

```diff
@@ -130,7 +130,12 @@ private String getMessageWithArgs(String resourceBundle, final Locale locale, fi
                 return FormatUtils.formatMsg(source, LocaleUtility.getSourceLocale(), args);
             }
         }
-
+        if(locale == null){
+            throw new VIPJavaClientException(ConstantsMsg.LOCALE_CANNOT_NULL);
+        }
+        if(component == null || "".equalsIgnoreCase(component)){
```

Comment:
the component name is case sensitive

Reply:
When component's value is empty, it doesn't matter to use case sensitive or insensitive method when compare.